.circleci/config.yml

```diff
@@ -131,17 +137,19 @@ jobs:
             sudo chown -R circleci:circleci /cncf/
       - restore_cache:
           key: terraform-{{.Environment.CIRCLE_WORKFLOW_ID}}
-      - run: cp -a /home/circleci/project/terraform/terraform/* ./scripts/terraform/
-      - run: ssh-add /home/circleci/project/terraform/sshkey
+      - run: cp -a /home/circleci/project/terraform/terraform/* ./scripts/terraform/ || true
+      - run: ssh-add /home/circleci/project/terraform/sshkey || true
```

Comment:
Interesting why we need the `|| true` here?

Reply:
There is no terraform stuff in GKE case, alternative is to have if [] or move this logic into packet, script, probable it could be better idea to do.

Reply:
We can have a command. I'll optimise this after the merge. It is fine for now.